### DISCUSSION TITLE
want to use multi select and other questions

### DISCUSSION THREAD

**Discussion by taholmes160** [Budibase/budibase#8998] [Help]
taholmes160: In my database, I am tracking inventory of used cars, and the moves between sales sites that they make.  This is built on MySQL running on an azure account.
So i have units table that describes the cars, I have origin and destination tables for where they come from and where they go (These are relationship linked to the moves table) the moves table lists all the moves that a car makes over its time in our system.  I would like to have the ability to "request" a move of a car (or a group of cars) so that the sales manager can set up a batch of cars to move from one sale site to another, and the ability for the lead driver on the move to update the date the move was made and who drove each car.  my question is a couple part.  (1) how should I structure the tables to handle both the requested moves and the actual moves.  (2) how do I set up budibase to give me a filtered list of all the cars (either all cars or a given subset such as all the cars at sale site 1) that I can then "batch" update / tag /whatever to set them as requested to move -- (3) be able to update each car record to change to MOVED (thereby putting it in inventory at the new sale site, and taking it off the requested move list) This would be an individual operation b/c each car has a different driver on a given day (this is for tracking any damage that might occur) 

I can show you tables etc if you need -- While this is business application, right now its a hobby learning exercise for me (which I'm hoping my company will eventually pay me for, but not yet)  I really need some pointers on the underlying design as well as the budibase implementation (More the underlying design)

thanks
TIM

**Comment by andz-bb** [comment]
hey @taholmes160, could you send me the schema for your tables and possibly an app export if you already have some of this built please? I can have a look and see if we can come up with a solution for your use case. Thanks :)

**Comment by taholmes160** [comment]
Sure -- heres the export

https://drive.google.com/file/d/1hXvpb3SaPiT-rfVmbSXC8f6lrAqsoZHA/view?usp=sharing

Let me know what else you need -- im taholmes160 on discord in the budibase
server

Tim
